composer/modules/web/src/ballerina-theme/theme/elements/button.overrides

```diff
@@ -29,4 +29,16 @@
 .ui.inverted.form button {
     background: #f8f9fa;
     color: #626262;
-}
\ No newline at end of file
+}
+.debug-buttons{
```

Comment:
This does not belong in theme files. Please move these styles to something like debugger.scss